deploy/helm/kubecf/charts/eirini-extensions/templates/ssh-proxy.yaml

```diff
@@ -131,7 +131,13 @@ spec:
     protocol: TCP
     port: 2222
     targetPort: 2222
-  type: ClusterIP
-  {{- if gt (len .Values.externalIPs) 0 }}
+  type: {{- with index .Values $component }}
+  {{- if gt (len .externalIPs) 0 }}
   externalIPs: {{ .Values.externalIPs | toJson }}
   {{- end }}
+  {{- if .clusterIP }}
+  clusterIP: {{ .clusterIP | quote }}
+  {{- end }}
+  {{- if .loadBalancerIP }}
+  loadBalancerIP: {{ .loadBalancerIP | quote }}
+  {{- end }}
```

Comment:
I copied the logic from here: https://github.com/cloudfoundry-incubator/kubecf/blob/master/deploy/helm/kubecf/templates/ingress.yaml#L178

I need to try this out.

Reply:
Works on gke